app/models/concerns/aliquotable.rb

```diff
@@ -34,5 +34,23 @@ module Aliquotable
     has_many :derived_aliquots, -> { where(aliquot_type: :derived) },
              as: :source, class_name: 'Aliquot',
              dependent: :nullify, inverse_of: :source
+
+    # Method to calculate the used volume
```

Comment:
I think a bit more documentation would be nice. e.g. the used_volume is the sum of all of the volumes of the derived aliqouts. Also adding parameters and return values as per the standard heading documentation.